tools/psa/release.py

```diff
@@ -298,16 +317,10 @@ def get_parser():
 
 
 def prep_build_dir():
-    """
-    Creates a clean BUILD directory
-    """
     build_dir = os.path.join(ROOT, 'BUILD')
-    if os.path.exists(build_dir):
-        logger.debug("BUILD directory already exists... Deleting")
-        shutil.rmtree(build_dir)
-
-    os.makedirs(build_dir)
-    logger.info("BUILD directory created in {}".format(build_dir))
+    if not os.path.exists(build_dir):
```

Comment:
i believe this change is for incremental builds, if that's the case add a `-c` option to the argument parser for a clean build

Reply:
No, BUILd part for the specific target is still deleted as the current behavior.
This patch just avoid to delete other targets.